hip-more-cultural-names/common/landed_titles/0_HIP_MoreCulturalNames.txt

```diff
@@ -6971,33 +7879,27 @@ e_hre = {
             }
         }
         d_aquileia = {
-            arpitan = "Aquil�e" # Historical? Copied from Frankish
-            bosnian = "Oglej" # Historical? Copied from Carantanian
-            bulgarian = "Oglej"
-            carantanian = "Oglej"
-            croatian = "Oglej" # Historical? Copied from Carantanian
```

Comment:
why are you deleting those?

Reply:
d_aquileia is Friuli, as far as I found Oglej stands for Aquileia not Friuli